modules/Hoot/managers/api.js

```diff
@@ -146,6 +147,22 @@ export default class API {
 
     }
 
+    getDefaultOverpassQuery() {
+        if ( this.defaultOverpassQuery) {
+            return Promise.resolve( this.defaultOverpassQuery );
+        } else {
+            const params = {
+                path: '/grail/getDefaultOverpassQuery',
+                method: 'GET'
+            };
+            let that = this;
+            return this.request( params ).then( resp => {
```

Comment:
`.then(...` on new line for syntax consistency with rest of our codebase

Reply:
as you wish